plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -502,124 +501,6 @@ class FeatureDevController(
         messenger.sendAsyncEventProgress(tabId = tabId, inProgress = false)
     }
 
-    private suspend fun onCodeGeneration(session: Session, message: String, tabId: String) {
```

Comment:
Why is this being moved to another file ?

Reply:
Becuase it makes the testing easier, I'm testing the specific logic of this function in other unit tests below. In addition, it also makes the controller cleaner and easier to read.